freqtrade/tests/rpc/test_rpc.py

```diff
@@ -11,8 +11,8 @@
 from freqtrade.persistence import Trade
 from freqtrade.rpc import RPC, RPCException
 from freqtrade.state import State
-from freqtrade.tests.test_freqtradebot import (patch_coinmarketcap,
-                                               patch_get_signal)
+from freqtrade.tests.test_freqtradebot import patch_get_signal
```

Comment:
should we move patch_get_signal to conftest too? or is that not worth it as it's only user here other than that function?

Reply:
Nah, it was more to do with the fact that I'd like to get rid of it, or make it part of a fixture that gives you the ready bot or something like that. Still plenty of things I want to do, our tests are way too big and repeat so much stuff.